### ISSUE TITLE
about the two timepad

### ISSUE THREAD

**Issue by baiyizhourobert** [nthquan35/CMPT479#1]
hi, sorry for bothring your time. about the assignment 2 the two time pad function. how can u get the first petext0, re u using the cribdrag one by one to test?

**Comment by nthquan35** [comment]
yeah essential spent sleepless nights to guess letter by letter. Until you guess at least 2-3 words, you can google them out! GLHF

**Comment by baiyizhourobert** [comment]
sorry do u mind i add your discord or something else for more detail? i really appreciate your help

**Comment by nthquan35** [comment]
I took this class a year ago. I doubt it that I can recall anything from it. I would not be able to help I apologize.
